GETTING_STARTED.md

```diff
@@ -0,0 +1,374 @@
++    const todo = await drivers.http('TODO').post(
++      'https://todo-backend-modern-js.herokuapp.com/todos',
++      {
++        title: 'Yo! I am a new todo!',
++      },
++    )
++
++    store.todos.add(todo)
++  })
+]
+
+const AppContainer = listen(listeners)(App)
+
+export default provider(store)(AppContainer)
+```
+
+First we add a new driver to **k-ramel**, the http driver. (a.) \
+Then we change the `onClick` callback so it dispatchs an empty action whose type is `@@ui/ADD_TODO>CLICKED` (b.), this action is catched into our **listeners**, and we react to it by calling an API (c.).\
+The TODO title is hardcoded, you can see we `await` the API response, and add the todo to the store after the API responds.
+
+If you open your redux-devtools you will see that the HTTP driver trigger 2 actions (2 events) :
+ - One at start
+ - One at end
+
+We'll use this last event to trigger the addition to the store instead of awaiting the driver.
+
+## 8. Use the HTTP driver ENDED action
+```diff
+import React from 'react'
+import { createStore, types, when } from 'k-ramel'
+import { provider, inject, listen } from '@k-ramel/react'
+import createHttpDriver from '@k-ramel/driver-http'
+
+const store = createStore(
+  {
+    todos: types.keyValue(),
+  },
+  {
+    drivers: {
+      http: createHttpDriver(),
+    },
+  },
+)
+
+store.todos.add({ id: 2, title: 'finish the documentation' })
+store.todos.add({ id: 32, title: 'an other' })
+
+console.log(store.todos.get(2))
+
+const Todos = ({ todos, onClick }) => (
+  <div>
+    <button onClick={onClick}>Add a todo!</button> {/* c. */}
+    <ul>
+      {todos.map(({ id, title }) => (
+        <li key={id}>{title}</li>
+      ))}
+    </ul>
+  </div>
+)
+
+const TodosContainer = inject((store) => {
+  return {
+    todos: store.todos.getAsArray(),
+    onClick: () => store.dispatch('@@ui/ADD_TODO>CLICKED')
+  }
+})(Todos)
+
+const App = () => (
+  <TodosContainer />
+)
+
+const listeners = [
+  when('@@krf/ADD>TODOS')(() => {
+    console.log('A todo is added!')
+  }),
+-  when('@@ui/ADD_TODO>CLICKED')(async (action, store, drivers) => {
+-    const todo = await drivers.http('TODO').post(
++  when('@@ui/ADD_TODO>CLICKED')((action, store, drivers) => {
++    drivers.http('TODO').post(
+      'https://todo-backend-modern-js.herokuapp.com/todos',
+      {
+        title: 'Yo! I am a new todo!',
+      },
+    )
+-
+-    store.todos.add(todo)
+  }),
++  when('@@http/TODO>POST>ENDED')((action, store) => {
++    store.todos.add(action.payload)
++  }),
+]
+
+const AppContainer = listen(listeners)(App)
+
+export default provider(store)(AppContainer)
+```
+
+We don't use **async/await** anymore but we use Redux as an eventbus and react to the **ENDED** event triggered by the HTTP driver to add the todo into the store.
+
+You can find the last version here in [examples/getting_started](https://github.com/alakarteio/k-ramel/tree/master/examples/getting_started).
```

Comment:
```suggestion
You can find this last step in this project : [examples/getting_started](https://github.com/alakarteio/k-ramel/tree/master/examples/getting_started).
```

Reply:
@frinyvonnick What is your though about it?

@delphinemillet yours?

Reply:
This is about this comment: https://github.com/alakarteio/k-ramel/pull/314#discussion_r253266811

Reply:
may be you should target a specific line or at least a specific file in this folder ?